modules/network_synchronizer/networked_controller.cpp

```diff
@@ -637,7 +637,7 @@ void ServerController::receive_inputs(Vector<uint8_t> p_data) {
 
 	const uint32_t now = OS::get_singleton()->get_ticks_msec();
 	// If negative the timer was disabled, so just assume 0.
-	network_watcher.push(MAX(0, now - input_arrival_time));
```

Comment:
I feel like this line is simpler though

Reply:
You right, the type is unsigned no way this can be negative. This change is good, can you please just fix the comment too?